js/viewmodel.js

```diff
@@ -157,7 +157,12 @@ window.FBE = window.FBE || {};
                     },
                     rawEntities[name],
                     // fix bad data in factorio-blueprint
-                    name === 'steam_turbine' && {width: 5, height: 3}                 
+                    name === 'steam_turbine' && { width: 5, height: 3 },

```

Comment:
can we move these into a switch statement? I found this really hard to understand what it was doing.